### ISSUE TITLE
Final project peer review

### ISSUE THREAD

**Issue by yangdanny97** [Deep14/Movies-Analysis#13]
The project's goal was to predict a movie's revenue from information about its production and cast. The data was drawn from a Kaggle dataset and information scraped from IMDB.

The preprocessing and feature selection was well thought out and well explained. I especially liked how the team accounted for inflation and the binary encoding format of the genres. I also liked the incorporation of data scraped from the web, and would have liked to see more web-scraped features included in the dataset. 

I have some concerns about transforming the budget and revenue though, because I feel that what seems like even small errors would actually have massive implications for usefulness of your prediction (getting the log off by 1 is the same thing as your prediction being off by a factor of e). Thus, loss functions would need to harshly penalize even small errors, and I feel like the methods used may not have taken this into account.

Your methods and results for the most part are pretty well explained, but I had a hard time figuring out what the formulas meant or how they related to the loss functions you were using. I also would have liked to see how you tuned the parameters for your decision tree and its final structure. The neural network was interesting but an R^2 of 0.1 means that it's practically uncorrelated. Perhaps it could have done better with more tuning, and I would have liked to see more detail on how you trained the neural net and why you only had a single hidden layer.

From reading your results it seems like the features that you were using might not have captured all of the important predictors of a movie's revenue. A few ideas for features that might have resulted in better fitting models: advertising budget (or what percentage of the budget was spent on advertising) and the number of theaters screening the movie. 

Overall though, great work I really enjoyed reading your report. :)
